src/kagglehub/handle.py

```diff
@@ -83,6 +83,25 @@ def to_url(self) -> str:
         return base_url
 
 
+@dataclass
+class CodeHandle(ResourceHandle):
+    owner: str
+    notebook: str
+    version: Optional[int] = None
+
+    def is_versioned(self) -> bool:
+        return self.version is not None and self.version > 0
+
+    def __str__(self) -> str:
+        handle_str = f"{self.owner}/{self.notebook}"
```

Comment:
You should include the version in the __str__ and to_url if it the user specifies a specific version of the kernel.

Reply:
I see the parse method below doesn't support versioned handles, are you planning to support it?

If not, you can remove `version` and `is_versioned` from `CodeHandle`.